tests/h/session_test.py

```diff
@@ -69,8 +69,28 @@ def test_model_show_sidebar_tutorial(
         assert 'show_sidebar_tutorial' not in preferences
 
 
+def test_profile_userid_unauthenticated(unauthenticated_request):
```

Comment:
It'll be good to expand the unit tests a bit. There are tests for all keys in the model tests which we should probably duplicate. Because the `model` function will probably be deleted in the coming months, so just duplicating the tests would make the process of deleting that one easier.